utils/appliance.py

```diff
@@ -52,12 +52,14 @@ class IPAppliance(object):
     it knows both the provider, vm_name and can there for derive the IP address.
 
     Args:
-        ipaddress: The IP address of the provider
-        browser_streal: If True then then current browser is killed and the new appliance
+        address: The IP address of the provider
+        browser_steal: If True then then current browser is killed and the new appliance
             is used to generate a new session.
+        ssh_port: ssh port of the provider
+        db_port: database port of the provider
```

Comment:
`s/provider/appliance/` when you are here, I don't know why is washere.

Possibly, the ssh and db port should be also passed in the Appliance class which gets vm name and provider.